semanticcpg/src/main/scala/io/shiftleft/semanticcpg/language/types/structure/LiteralTraversal.scala

```diff
@@ -0,0 +1,9 @@
+package io.shiftleft.semanticcpg.language.types.structure
+
+import io.shiftleft.codepropertygraph.generated.nodes.Literal
+import io.shiftleft.semanticcpg.language.toLiteralMethods
+
+class LiteralTraversal(val traversal: Iterator[Literal]) extends AnyVal {
+  def innerText: Iterator[Option[String]] =
```

Comment:
I think the ideal behaviour is that this should always be a `String`, so for non-string literals we just return `.code` basically. Is that correct, @ml86  or do you prefer that this returns an `Option`?

Reply:
Would we want the same behavior on the `.innerText` function in `LiteralMethods`?